mitodl/odl-video-service
# Dorska/collection pagination frontend

#### What are the relevant tickets?
#523 

#### What's this PR do?
Limits to 10 collections in sidebar.

#### How should this be manually tested?
1. Ensure that you have > 10 collections.
2. Ensure that in `.env`,  your `PAGE_SIZE_COLLECTIONS` is > 10 . (remember to restart if you change the `.env` file)
3. Go to any page and open the side bar. You should see only 10 collections.
